src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -456,21 +485,24 @@ public ReadOnlyCollection<T> ToReadOnlyCollection()
                 items = ToArray();
             }
             _items = Array.Empty<T>();
-            _size = 0;
+            Count = 0;
             _version++;
 
             return new TrueReadOnlyCollection<T>(items);
         }
 
+        // Ensures that the capacity of this list is at least the given minimum
+        // value. If the currect capacity of the list is less than min, the
+        // capacity is increased to twice the current capacity or to min,
+        // whichever is larger.
```

Comment:
I knows english only a little so I have not enough competitions to write comments. This comment I coped from List<T>